superset/migrations/migration_utils.py

```diff
@@ -0,0 +1,37 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from alembic.operations import BatchOperations, Operations
+
+
+def create_unique_constraint(
```

Comment:
I know we had some trouble in the past with some databases where we couldn't delete the constraint because the constraint name couldn't be inferred. Would it be possible to explicitly pass a constraint name here and in `drop_unique_constraint` to prevent this from happening again?

Reply:
I remember seeing this before too, and three's also some instance of us using a `naming_convention` parameter in alembic. I thought about using this feature but wasn't clear on how it would affect things... Let me try something

Reply:
I think using the `naming_convention` feature should do the trick, it's in the latest commit. https://alembic.sqlalchemy.org/en/latest/ops.html#alembic.operations.AbstractOperations.batch_alter_table.params.naming_convention